lib/ea/address_lookup/adapters/address_facade.rb

```diff
@@ -82,6 +82,7 @@ def http_get(path, query_params = {})
             method: :get,
             url: http_address,
             proxy: false,
+            timeout: EA::AddressLookup.config.timeout_in_seconds,
```

Comment:
I'd add a private `config` method:

```ruby
def config
  @config ||= EA::AddressLookup.config
end
```
Then you can just call `config.timeout_in_seconds`, and refactor the other called to config to suit.